src/js/structs/Job.js

```diff
@@ -0,0 +1,21 @@
+import Item from './Item';
+
+module.exports = class Job extends Item {
+  constructor() {
+    super(...arguments);
+
+    const id = this.getId();
+    if (id !== '/' && (!id.startsWith('/') || id.endsWith('/'))) {
+      throw new Error(`Id (${id}) must start with a leading slash ("/") and should not end with a slash, except for root id which is only a slash`);
```

Comment:
@mlunoe 
I think it's not a good idea to force this and throw an error.
There is this small possibility that the API maybe returns an ID with a slash at the end.
In this case our whole application will fail now. 
Also I think the `id` shouldn't validated in the constructor of `Job`. I don't expect it at this point.